### PR TITLE
Tm 91 auditing

### PR DESCRIPTION
**Pull request by chrisfoster76**

### CODE REVIEW

File: src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/UserService/IUserService.cs [hunk 1]
```diff
@@ -7,5 +7,9 @@ namespace SFA.DAS.LevyTransferMatching.Infrastructure.Services.UserService
     public interface IUserService
     {
         Task<IEnumerable<UserAccountDto>> GetLoggedInUserAccounts();
+
+        string GetUserId();
+
+        string GetUserDisplayName();
```
Comment (mmiddleton3301) [inline src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/UserService/IUserService.cs:13]: In the APIM review, I mentioned we could probably put all of this kind of stuff into one model that gets passed around - given the two _are_ related, you could combine these two methods into one.

**Review by mmiddleton3301 — COMMENTED** [review]
Just one point, relating to another on the APIM PR.

File: src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs [hunk 2]
```diff
@@ -120,18 +121,20 @@ public async Task<string> SubmitPledge(CreatePostRequest request)
 
             ValidateCreatePledgeCacheItem(cacheItem);
 
-            var pledgeDto = new PledgeDto
+            var apiRequest = new CreatePledgeRequest
```
Comment (DannyGB) [inline src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs:124]: I might call this variable `createPledgeRequest` to reflect its type

**Review by DannyGB — APPROVED** [review]

File: src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs (hunk 2 above)
Comment (chrisfoster76) [reply src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs:124]: Renamed

**Pull request merged**